### ISSUE TITLE
[Support] Will unilaterall close (force-close) of the Eclair-created channel from the remote (LND) side make funds appear on regular on-chain address of the Eclair-generated seed?

### ISSUE THREAD

**Issue by Talkless** [ACINQ/eclair-mobile#258]
Relative of mine had Eclair wallet with some funds on-chain and some on LN channel.

That phone is now dead.

He has recovery phrase, recovered into another (BlueWallet, iOS) wallet, but sees on-chain funds only (obviously).

LN Google Drive backup is not found in Eclair installed on Android VM - probably forgot to enable Google Drive backup...

Now, useful fact is that his LN funds are locked in the channel with _my_ LND node.

Question: if I perform force-close (getting my part after 2016 blocks) from LND side, will his balance will be sent to his "regular" on-chain address that will be visible on that other on-chain wallet? Or that force-close address is some sort specific, "LN-related", from different derivation path (or whatever)?

Thanks!

**Comment by t-bast** [comment]
Restoring the seed in a different lightning wallet will not work for off-chain funds.
See https://github.com/ACINQ/eclair/blob/b75f6c36e9bab8b77702e16f2514d9accc47cd8d/eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala#L183 for recovering the on-chain funds.

To recover the off-chain funds, you need to restore the seed in eclair-mobile, that will tell your peer to close the channel and you'll get the funds back on-chain.

**Comment by t-bast** [comment]
If you do the force-close from your lnd node, the result will be the same.
But your friend will need an eclair-mobile running to correctly claim his output of the commitment tx to his "regular" on-chain address.

**Comment by t-bast** [comment]
> LN Google Drive backup is not found in Eclair installed on Android VM

Note that if there's really no backup recovery possible at all, you  may need to use https://github.com/ACINQ/eclair-recovery-tool to recover the main output.

**Comment by dpad85** [comment]
> LN Google Drive backup is not found in Eclair installed on Android VM - probably forgot to enable Google Drive backup...

Did he check his Google Drive folder? Backup will be in a folder named `eclair-mobile` in his Drive documents :

![image](https://user-images.githubusercontent.com/5765435/103206295-d4da4e80-48fb-11eb-988b-388a05076a2e.png)

**Comment by Talkless** [comment]
> ....to correctly claim...

OK so that's what I wanted to know - you need to perform one more tx to claim. That obviously needs LN info..

> you may need to use https://github.com/ACINQ/eclair-recovery-tool to recover the main output.

That link does not work and searching manually does not help.

**Comment by t-bast** [comment]
> That link does not work and searching manually does not help.

Woops you're right, that's a branch in eclair at the moment: https://github.com/ACINQ/eclair/pull/1197

Can you double-check the google drive backup? That would be the easy solution to recover the funds, otherwise we'll look into the recovery tool option.

**Comment by Talkless** [comment]
> Can you double-check the google drive backup? 

Yes, he said he is completely positive that there's no Eclair backup. He will try building Eclair after exams.
